README.md

```diff
@@ -72,6 +72,16 @@ See below for how to configure the environment variables appropriately
 - Awattar__VATMultiplier=1.00 # Optional (default: 1.19), you should not need to set this unless your VAT differs from the default
 ```
 
+### Barry
```

Comment:
You'll want to edit the list of supported energy providers / tariffs near the top of this file to include your new one